alf/trainers/policy_trainer.py

```diff
@@ -294,20 +389,7 @@ def _train(self):
                 common.write_gin_configs(self._root_dir, "configured.gin")
 
                 with alf.summary.record_if(lambda: True):
-
-                    def _markdownify(paragraph):
-                        return "    ".join(
-                            (os.linesep + paragraph).splitlines(keepends=True))
-
-                    common.summarize_gin_config()
-                    alf.summary.text('commandline', ' '.join(sys.argv))
-                    alf.summary.text(
-                        'optimizers',
-                        _markdownify(self._algorithm.get_optimizer_info()))
-                    alf.summary.text('revision', git_utils.get_revision())
-                    alf.summary.text('diff',
-                                     _markdownify(git_utils.get_diff()))
-                    alf.summary.text('seed', str(self._random_seed))
+                    self._set_summary()
```

Comment:
line 387-391 can also be put into this function

Reply:
This has been addressed following you previous comments. Seems that github is somehow showing you an older commits?